src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -2,46 +2,50 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.IO;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Html;
 using Microsoft.Extensions.Internal;
 
 namespace Microsoft.AspNetCore.Razor.TagHelpers
 {
     /// <summary>
     /// An HTML tag helper attribute.
     /// </summary>
-    public class TagHelperAttribute
+    public class TagHelperAttribute : IHtmlContentContainer
```

Comment:
Should be no need for this class to implement `IHtmlContentContainer`. `TagHelperOutput` knows what this class exposes.